### DISCUSSION TITLE
Error Setting Firestore/Storage Rules when Connecting Rowy to Firebase

### DISCUSSION THREAD

**Discussion by hanmangt** [AntlerVC/firetable#1457] [Support / Q&A] [answered]
We're in the process of setting up a new project and we're encountering an issue while attempting to connect Rowy with our Firebase. The specific error message we're receiving is: "Error setting Firestore/Storage rules: Requested entity was not found."

We've checked that the Firebase rules are configured. Any ideas on what we might have missed to make the Rowy connection work?
![image](https://github.com/rowyio/rowy/assets/149450909/d02981ee-2896-4b2c-8460-5bc7611298f2)

**Comment by bofeiw** [comment, marked as answer]
Hi, we have found a backend issue that causes this error, however a retry should completes this step.

If you have already completed the setup but you see access denied, you will need to check if your Firestore rules are set to this template: https://docs.rowy.io/how-to/roles#write-firestore-security-rules-with-user-roles.

Sorry for the inconvinience, we will fix the setup issues soon.
